app/views/marketing/marketing.html.erb

```diff
@@ -172,6 +172,11 @@
       <span><img src="<%= image_path 'marketing/trusted--mozilla.png' %>" alt="Mozilla"></span>
       <span><img src="<%= image_path 'marketing/trusted--adobe.png' %>" alt="Adobe"></span>
 		</div>
+    <br>
```

Comment:
Yeah gonna remove the `<br>` forsure, it was just to preview ill CSS it. Mix of tabs and spaces I didn't see in my IDE which is weird